Foundation/NSISO8601DateFormatter.swift

```diff
@@ -10,70 +10,111 @@
 import CoreFoundation
```

Comment:
nit: Since the class name does not have the "NS" prefix, it may make sense to drop it from the filename as well